.github/workflows/container-image.yml

```diff
@@ -84,6 +96,17 @@ jobs:
           outputs: type=docker,dest=/tmp/kubewarden-controller-image-${{ env.TAG_NAME }}.tar
           tags: |
             ghcr.io/${{github.repository_owner}}/kubewarden-controller:${{ env.TAG_NAME }}
+      -
+        name: Create SBOM file
+        shell: bash
+        run: |
+          bom generate -n https://kubewarden.io/kubewarden.spdx -o kubewarden-controller.spdx .
+      -
+        name: Attach SBOM file in the container image
+        shell: bash
+        run: |
+          set -e
+          cosign attach sbom --sbom kubewarden-controller.spdx "ghcr.io/${{github.repository_owner}}/kubewarden-controller:${{ env.TAG_NAME }}"
```

Comment:
Can we use the digest from the output of `build-push-action` instead of the tag?

Reply:
I see you've just done it. Thanks!

Reply:
Yes, I was working on that since your first review. Please take a look again. I just pushed the change.